packages/odyssey-storybook/src/guidelines/Typography-MUI.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from "@mui/material";
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { ThemeProvider } from "@storybook/theming";
+import { theme } from "@okta/odyssey-react-mui";
+
+import { Meta, Canvas } from "@storybook/addon-docs";
+
+<Meta title="Guidelines/MUI Typography" />
+
+# Typography
+
+A set of pre-defined text styles for headers, body copy, and links designed for readability and clarity of hierarchy.
+
+## Type styles
+
+<MuiThemeProvider theme={theme}>
```

Comment:
The decorator only applies to stories, so non-story MUI examples still need a wrapper. Once I have an example for storing MUI components directly, I can convert these examples to stories.